source/Quickstart-GKE.rst

```diff
@@ -0,0 +1,326 @@
+you have edited the statement, you may run the command in your local
+shell or in the Cloud Shell.
+
+For this document, we are using the local shell.
+
+::
+
+   gcloud container clusters get-credentials standard-cluster-1 --zone us-central1-a --project <project name>
+
+Installing the Operator
+-----------------------
+
+1. Use the ``git clone`` command to download the correct branch of the
+   percona-xtradb-cluster-operator repository.
+   **Note:** *In the git statement, you must specify the correct branch
+   with the -b option.*
+
+.. code:: bash
+
+       git clone -b release-0.3.0 https://github.com/percona/percona-xtradb-cluster-operator
+
+After the repository is downloaded, change the directory to run the rest
+of the commands in this document:
+
+::
+
+   cd percona-xtradb-cluster-operator
+
+2. Extensions of the Kubernetes API are custom resources. The Custom
+   Resource Definition for PXC adds the operator resources. This step is
+   only run once. After the Kubernetes API has been extended any
+   additional operator deployments use the same resources.
+
+.. code:: bash
+
+   kubectl apply -f deploy/crd.yaml
+
+The following confirmation is returned:
+
+::
+
+         customresourcedefinition.apiextensions.k8s.io/perconaxtradbclusters.pxc.percona.com created
+         customresourcedefinition.apiextensions.k8s.io/perconaxtradbbackups.pxc.percona.com created
+
+3. Create a namespace. The resource names must be unique within the
+   namespace and provide a way to divide cluster resources between users
+   spread across multiple projects.
+   You must replace the ``<namespace name>`` placeholder with a
+   descriptive name.
+
+.. code:: bash
+
+   kubectl create namespace <namespace name>
+
+The following confirmation is returned:
+
+::
+
+       namespace/<namespace name> created
+
+4. Set the context for the namespace, which saves the namespace for
+   subsequent commands. You must replace the ``<namespace name>``
+   placeholder with the name used in the earlier step.
+
+.. code:: bash
+
+   kubectl config set-context $(kubectl config current-context) --namespace=<namespace name>
+
+A successful return statement prints the project name, location (for
+this example, a zone), and cluster name.
+
+.. code:: bash
+
+   Context "<project name>_<zone location>_<cluster name>" modified.
+
+5. You use Cloud Identity and Access Management (Cloud IAM) to control
+   access to the cluster. You must have the ability to create Roles and
+   RoleBindings. In the following command, you must replace the email
+   address placeholder with your email address.
+
+.. code:: bash
+
+   kubectl create clusterrolebinding cluster-admin-binding \
+   --clusterrole cluster-admin --user email.address
+
+The return statement confirms the creation.
+
+.. code:: bash
+
+   clusterrolebinding.rbac.authorization.k8s.io/cluster-admin-binding created
+
+6. The role-based access control (RBAC) for the cluster should be
```

Comment:
for GKE CRD, RBAC, Operator steps can be simplified to one command
```
kubectl apply -f deploy/bundle.yaml
```

So, in result installation will can look like
```
gcloud container clusters create ...
kubectl create clusterrolebinding cluster-admin-binding ...
kubectl create namespace ...
kubectl config set-context ...
git clone ...
kubectl apply -f deploy/bundle.yaml
kubectl apply -f deploy/secrets.yaml
kubectl apply -f deploy/cr.yaml
```